server/bin/unittests

```diff
@@ -270,12 +268,14 @@ function _setup_state {
     let res=res+$?
     cp -a $_tdir/{unittests,getconf.py,pbench*} $_testopt_sat/bin
     let res=res+$?
-    cp -a $_tdir/../../lib/* $_testopt/common/lib
-    let res=res+$?
     cp -a $_tdir/../lib $_testopt
     let res=res+$?
+    cp -a $_tdir/../../lib/* $_testopt/lib
```

Comment:
This creates a requirement that no file in `$_tdir/../lib/` have the same name as any file in `$_tdir/../../lib/`, or it requires the understanding that any file in `$_tdir/../lib/` will be overridden by a file in `$_tdir/../../lib/` if they have the same name.

At the very least, this strikes me as odd -- shouldn't the "more local" files in `$_tdir/../lib/` be overriding the files in `$_tdir/../../lib/`?

Reply:
Ditto for lines 275 & 277.

Reply:
This is already the case, because this is what the `server/Makefile` does.

This change was made so that I could feel a bit more confident that the unit tests were operating closer to what the RPM based install does.